atlas-nagios-check/.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ripe.atlas.sagan"]
```

Comment:
This might be better accomplished by turning this whole thing into a proper python module with a `setup.py` and everything.  That way you can just include Sagan as a dependency and people do `pip install ripe.atlas.nagios` or something like that.